lib/src/util/stack_trace_mapper.dart

```diff
@@ -3,84 +3,81 @@
 // BSD-style license that can be found in the LICENSE file.
 
 import 'dart:async';
+import 'package:collection/collection.dart';
+
```

Comment:
This isn't where the newline should go. See [Effective Dart](https://www.dartlang.org/guides/language/effective-dart/style#do-place-dart-imports-before-other-imports).